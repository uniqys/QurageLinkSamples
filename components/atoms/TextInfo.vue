<template>
  <p class="has-text-success">
    <slot />
  </p>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator'

@Component
export default class TextInfo extends Vue {}
</script>
