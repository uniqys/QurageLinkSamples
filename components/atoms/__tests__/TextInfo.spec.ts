import { shallowMount } from '@vue/test-utils'
import TextInfo from '~/components/atoms/TextInfo.vue'

describe('TextInfo.vue', () => {
  it('should to match snapshot', () => {
    const text = 'test text'
    const wrapper = shallowMount(TextInfo, {
      slots: {
        default: text
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
