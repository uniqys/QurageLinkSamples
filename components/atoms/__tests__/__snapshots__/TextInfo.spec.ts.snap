// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInfo.vue should to match snapshot 1`] = `
<p
  class="has-text-success"
>
  test text
</p>
`;
